compiler/testData/codegen/bytecodeText/forLoop/forInUntil/forInUntilIntMinValue.kt

```diff
@@ -17,10 +17,10 @@ fun f(a: Int): Int {
 // 0 getFirst
 // 0 getLast
 // 0 getStep
-// 0 LINENUMBER 7
```

Comment:
Why is this change needed? Looks like a regression...

Reply:
Firstly, there is no code in line 7. It used to be correct, but it was not adjusted to line 6 in https://github.com/JetBrains/kotlin/commit/496765f41e26ea05a203a92e3ac39fea08e6604e#diff-eb501130ee852898f2efdc4789134b52

Also, the loop is only eliminated as dead code in JVM_IR. So `0 LINENUMBER 6` is only relevant for JVM_IR.

Reply:
Thanks, makes total sense.